intl/msg_hash_pl.h

```diff
@@ -3830,9 +3830,202 @@ MSG_HASH(MENU_ENUM_LABEL_VALUE_MENU_XMB_ANIMATION_OPENING_MAIN_MENU,
       "Menu główne animacji Otwiera/Zamyka")
 MSG_HASH(
     MENU_ENUM_LABEL_VALUE_DISC_INFORMATION,
-    "Disc Information"
+    "Informacje o dysku"
     )
 MSG_HASH(
     MENU_ENUM_SUBLABEL_DISC_INFORMATION,
-    "View information about inserted media discs."
+    "Wyświetl informacje o włożonych dyskach multimedialnych."
+    )
+   MSG_HASH(
+    MENU_ENUM_LABEL_VALUE_QUICK_MENU_SHOW_SET_CORE_ASSOCIATION,
+    "Pokaż Set Core Association"
+    )
+MSG_HASH(
+    MENU_ENUM_SUBLABEL_QUICK_MENU_SHOW_SET_CORE_ASSOCIATION,
+    "Pokaż/ukryj opcję „Set Core Association”."
+    )
+   MSG_HASH(
+    MENU_ENUM_LABEL_VALUE_QUICK_MENU_SHOW_DOWNLOAD_THUMBNAILS,
+    "Pokaż Pobierz Thumbnailson"
+    )
+MSG_HASH(
+    MENU_ENUM_SUBLABEL_QUICK_MENU_SHOW_DOWNLOAD_THUMBNAILS,
+    "Pokaż/ukryj opcję „Pobierz miniatury”."
+    )
+   MSG_HASH(
+    MENU_ENUM_LABEL_VALUE_MENU_SHOW_LEGACY_THUMBNAIL_UPDATER,
+    "Pokaż aktualizację starszych miniatur"
+    )
+MSG_HASH(
+    MENU_ENUM_SUBLABEL_MENU_SHOW_LEGACY_THUMBNAIL_UPDATER,
+    "Pokaż/ukryj możliwość pobierania starszych pakietów miniatur."
+    )
+   MSG_HASH(
+    MENU_ENUM_LABEL_VALUE_MENU_SHOW_QUIT_RETROARCH,
+    "Pokaż Uruchom ponownie RetroArch"
+    )
+MSG_HASH(
+    MENU_ENUM_SUBLABEL_MENU_SHOW_QUIT_RETROARCH,
+    "Pokaż/ukryj opcję „Uruchom ponownie RetroArch”."
+    )
+#else
```

Comment:
Try removing this line, this is the relevant error.

```
In file included from intl/msg_hash_pl.c:47:
intl/msg_hash_pl.h:3871:2: error: #else without #if
 #else
  ^~~~
```
I would also check where you copy pasted it from and see if you missed something?

Reply:
hmm but else is in US file

Reply:
Look for the matching `#if` or `#ifdef` and `#endif` in the us file.